src/main/scala/rise/OpenCL/DSL.scala

```diff
@@ -28,7 +28,7 @@ object DSL {
       MapWorkGroup(0)()
   }
 
-  def toMem: ToMem = ToMem()()
+  def toMem: OclToMem = OclToMem()()
```

Comment:
do we need to have this "Ocl" prefix?

Reply:
I vaguely remember that it was important to have different names in rise/the DSL because of annoying conflicts otherwise. In that context it made sense to also use different names in DPIA I think.